doc/getting_started_rtkernel.rst

```diff
@@ -80,6 +66,17 @@ following to create and configure the build::
     -DCMAKE_ECLIPSE_EXECUTABLE=/opt/rt-tools/workbench/Workbench \
     -DCMAKE_ECLIPSE_GENERATE_SOURCE_PROJECT=TRUE -G "Eclipse CDT4 - Unix Makefiles"
 
+If you are instead is using Windows, start an MSYS session, such as the program 
+"Command line" found in the rt-labs rt-collab Toolbox installation::
+
+    cd /c/Users/rtlfrm/Documents/Profinet/build_rtkernel
+    export RTK=/c/Users/rtlfrm/Documents/Profinet/rt-kernel
```

Comment:
export RTK=`<PATH TO rt-kernel>`

Reply:
Hm. In my experience, the tricky part when working in Windows is to get the paths correct (Should I write C: or /c/, or /mnt/c/, or something else? Will the tool accept escaped whitespace?). Writing <PATH TO X> will hide this tricky part.

Reply:
Ok, good point.
Maybe write "Adapt paths to your setup"?

Reply:
Done.